Adds support for new audit log events

Khaaz opened the pull request:
This updates eris according to discordapp/discord-api-docs#1191.

It adds:
```
MEMBER_MOVE: 26
MEMBER_DISCONNECT: 27
BOT_ADD: 28

MESSAGE_BULK_DELETE: 73
MESSAGE_PIN: 74
MESSAGE_UNPIN: 75

INTEGRATION_CREATE: 80
INTEGRATION_UPDATE: 81
INTEGRATION_DELETE: 82
```

This PR also updates the typings.

Khaaz committed:
Adds new audit logs event

Khaaz committed:
Update typings for new audit logs event

Khaaz committed:
Fix ; instead of ,

Khaaz committed:
Fix the linting issue

abalabahaha reviewed (changes requested):
`GuildAuditLogEntry#target` getter needs to be updated to return null for integration actions

Khaaz committed:
Updates GuildAuditLogEntry with new types.
Return null for target for 26 and 27.
Returns null for target for integrations events

Khaaz committed:
Merge branch 'dev' into dev

abalabahaha commented on lib/structures/GuildAuditLogEntry.js:
```diff
@@ -90,6 +90,9 @@ class GuildAuditLogEntry extends Base {
         } else if(this.actionType < 20) { // Channel
             return this.guild && this.guild.channels.get(this.targetID);
         } else if(this.actionType < 30) { // Member
+            if(this.actionType === 26 || this.actionType === 27) { // MEMBER_MOVE / MEMBER_DISCONNECT
```
Constants can be used here

Khaaz committed:
Replace hardocoded values to constants (member move/disconnect)

Khaaz committed:
Merge branch 'dev' of https://github.com/Khaazz/eris into dev
